gcloud/pubsub/client.py

```diff
@@ -50,6 +50,12 @@ def list_topics(self, page_size=None, page_token=None):
         See:
         https://cloud.google.com/pubsub/reference/rest/v1/projects.topics/list
 
+        Example:
+
+        .. literalinclude:: pubsub_snippets.py
+           :start-after: [START client_list_topics]
+           :end-before: [END client_list_topics]
```

Comment:
So let me make sure I understand. If we go this direction then...

1. `print client.list_topics.__doc__` will print an ugly and un-useful example (pointing people to go look at a big long file)
2. If I browse to the `pubsub.Client` and look at the `list_topics` method, I'll see a pretty example :).

Is that right?

I have to also ask... Could we flip this around the other way?

In other words... Could we have the snippet live officially in that docstring, and have the `pubsub_snippets.py` file pull in the example piece from the docstring?

Would that make it so that we get pretty examples in the docs for the `list_topics` method (2) as well as useful output when you type `help(client.list_topics)` ? (Along with the code being tested obviously...)